src/components/Form.js

```diff
@@ -5,15 +5,22 @@ import "../styles/components/form.scss";
 const Form = (props) => {
     const { title, paragraphs, textarea, link } = props.data;
 
+    const paragraphsContainer = paragraphs.map((paragraph) => (
+        <p className="form__paragraph" key={paragraph.slice(0, 33)}>
+            {paragraph}
+        </p>
+    ));
+    const checkLink = link ? (
```

Comment:
It would be nice to keep it as a separate component.
double & check if param is true-ish and in default as a second argument you get `null` if you want to have a boolean here you can go for `!!link && (..)` it means first `!` gives you an opposite value in the boolean, second `!` your original value in the boolean. It is super handy but here just use `&&` like that: 

```
const checkLink = link && (
        <a href={link.path} className="form__link">
            {link.content}
        </a>
    )
```

Reply:
Logical operator implemented.
I was thinking about making separate directory for controller files but this app is to small for this kind of logic. If it grow bigger then it might be implemented.